inc/storefront-template-functions.php

```diff
@@ -139,9 +139,9 @@ function storefront_credit() {
 
 		if ( apply_filters( 'storefront_credit_link', true ) ) {
 			if ( storefront_is_woocommerce_activated() ) {
-				$links_output .= '<a href="https://woocommerce.com" target="_blank" title="' . esc_attr__( 'WooCommerce - The Best eCommerce Platform for WordPress', 'storefront' ) . '" rel="noreferrer nofollow">' . esc_html__( 'Built with Storefront &amp; WooCommerce', 'storefront' ) . '</a>.';
+				$links_output .= '<a href="https://woo.com" target="_blank" title="' . esc_attr__( 'WooCommerce - The Best eCommerce Platform for WordPress', 'storefront' ) . '" rel="noreferrer nofollow">' . esc_html__( 'Built with Storefront &amp; WooCommerce', 'storefront' ) . '</a>.';
```

Comment:
Not a blocker but it was recommended to also update this text to read:

> Built with WooCommerce.

The reasoning was that it simplifies the text, and puts the emphasis on WooCommerce, with the URL pointing to our new landing page for WooCommerce.

See this thread for context: p1704919987116639/1704919648.335339-slack-C02PXDRC67P

Reply:
Ah I missed that. Updated now.

Reply:
Updated the language pot file as well.

Reply:
I also notice the E2E tests were failing because of the change to the text in the footer credit.

 I took the liberty to update the test (in be6292d) to look for the updated output.